visualization/eolearn/visualization/eopatch_base.py

```diff
@@ -95,7 +95,7 @@ def collect_and_prepare_feature(self) -> Tuple[Any, List[dt.datetime]]:
         data = self.eopatch[self.feature]
         timestamps = self.eopatch.timestamps
 
-        if feature_type.is_raster():
+        if feature_type.is_array():
```

Comment:
timestamps wont be a FeatureType for too long, so I'd not worry about that too much.
Since we have a `is_vector` i feel like it's counterpart should be `is_array` or perhaps `ìs_tabular`?

Reply:
i find array nice, tbh, I'm just missing something that would hint at multidimensionality. `is_matrix` pops to mind. but if you're not enthusiastic about it then let's leave it as it is

Reply:
to me matrices are almost always 3D, so if anything i'd use `ndarray` but i find `array` a bit nicer i guess